mcs/class/System/common.sources

```diff
@@ -147,18 +147,28 @@ System.Security.Authentication.ExtendedProtection.Configuration/ExtendedProtecti
 System.Security.Authentication.ExtendedProtection.Configuration/ServiceNameElement.cs
 System.Security.Authentication.ExtendedProtection.Configuration/ServiceNameElementCollection.cs
 
-System.Security.Cryptography/AsnEncodedData.cs
-System.Security.Cryptography/AsnEncodedDataCollection.cs
-System.Security.Cryptography/AsnEncodedDataEnumerator.cs
+../../../external/corefx/src/Common/src/System/Security/Cryptography/DerEncoder.cs
+../../../external/corefx/src/Common/src/System/Security/Cryptography/DerSequenceReader.cs
+../../../external/corefx/src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Helpers.cs
+../../../external/corefx/src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Oids.cs
+../../../external/corefx/src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ManagedX509ExtensionProcessor.cs
```

Comment:
I haven't looked at their PAL yet, but you are referencing a directory called `Pal.Unix` from `common.sources`.

If these classes are platform-specific, then they need to go into the relevant platform-specific `.sources` files and we need to make sure to have an implementation for our platforms.

Can we use this `Pal.Unix` on iOS and Android and what about Windows?

Reply:
It is located in Pal.Unix, but it is actually a fully managed implementation that works on all the platforms. On CoreFX it's used on all non-Windows platforms and a separate implementation is provided for Windows. It would make sense to eventually use the CoreFX Windows implementation on Windows, but it's not strictly necessary.

Reply:
Oh cool!  Maybe we could suggest to them to rename it into something like `Pal.Managed` to make it more obvious that it's not just for Unix?

Reply:
yeah, that code should not be under Pal.Unix/ in corefx

Reply:
It's kind of a common occasion in the X509 code that a managed implementation is present in the Pal.Unix / Pal.OSX directory. As part of https://github.com/dotnet/corefx/issues/30606 I would probably propose to rename couple of files to represent more accurately what it is supposed to do.